src/main/pages/che-7/overview/assembly_running-che-locally.adoc

```diff
@@ -69,32 +69,26 @@ Install the `chectl` command-line tool to be able to manage the Che Server appli
 
 === Installing chectl on Windows
 
-. Download link:https://github.com/che-incubator/chectl/releases/download/20190314232124/chectl-win.exe[chectl-win.exe].
-
-. Add the folder of the downloaded binary to your `PATH` environment variable.
-
-=== Installing chectl on macOS
-
-Run the following commands in the terminal:
-
+. Run the following commands in the powershell terminal:
++
 ----
-$ RELEASE="20190314232124/chectl-macos" && \
-URL=https://github.com/che-incubator/chectl/releases/download/${RELEASE} && \
-sudo curl -sSL ${URL} -o /usr/local/bin/chectl && \
-sudo chmod +x /usr/local/bin/chectl 
+C:\Users> Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://www.eclipse.org/che/chectl/win/'))
 ----
```

Comment:
maybe like for linux/mac, add:

The `chectl` binary is installed in `C:\ProgramData\chectl`.

Reply:
Thanks, I added it.